circle/specs-and-rake-tasks

```diff
@@ -25,8 +25,11 @@ echo "OK"
 RAILS_ENV=test rake db:schema:load
 RAILS_ENV=test JS_TEST=1 bundle exec teaspoon
 
+# Lint codebase 
+bundle exec rubocop
+
```

Comment:
I'm assuming that if any command returns with a non-zero value, CI fails and doesn't trigger a build?